lib/playground.dart

```diff
@@ -693,6 +705,22 @@ class Playground implements GistContainer, GistController {
     });
   }
 
+  Future _format() {
+    SourceRequest input = new SourceRequest()..source = _context.dartSource;
+
+    Future request = dartServices.format(input).timeout(serviceCallTimeout);
+    return request.then((FormatResponse result) {
+      busyLight.reset();
```

Comment:
Good suggestion about the toast. Done. If it turns out to be too loud and the button enablement is enough we can turn it off at a later date.

I haven't put in messaging if it fails for some reason, we'll be able to tell it through from diagnostics logs, I assume it will be very rare - if that turns out not to be the case we can revisit.